deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_crd.yaml

```diff
@@ -157,6 +157,13 @@ spec:
                   targetName:
                     description: Optional target name for registration
                     type: string
+              logging:
+                 description: Optional dynamic logging level configuration, version 8.9.5
```

Comment:
Suggest:

`Optional logging level configuration. Changing this value does not require restart of kubeturbo.`

Reply:
done